### PR TITLE
Responsivness

### PR DESCRIPTION
**Pull request by mohammed-sas**
[preview link](https://deploy-preview-13--mhd-racquets-store.netlify.app/)

The following were made mobile responsive
1.navbar
2.products listing page
3.cart page
4.wishlist page

PS
UI has to be improved

### CODE REVIEW

**Review by RakshaPawar108 — APPROVED** [review]
Great work Mohammed! Just a question: Can you please explain the purpose of app.module.css? I did not clearly understand its purpose and why you used it.

File: src/pages/products/products.module.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    grid-area: nav;
+}
+
+aside{
+    grid-area: aside;
+    overflow-y: scroll;
+    height: 100%;
+}
+
+main{
```
Comment (Jay-Gadhiya) [inline src/pages/products/products.module.css:31]: you can give some space after "main" for batter code readability.

File: src/pages/products/products.module.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+@media only screen and (max-width:900px){
+
+    .product-page-container{
+        height: 100%;
+        display: grid;
+        grid-template-rows:  1fr ;
+        grid-template-columns: 1fr;
+        grid-template-areas: " main";
+    }
+    main{
```
Comment (Jay-Gadhiya) [inline src/pages/products/products.module.css:69]: you can give space here also

File: src/app.module.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+nav{
+    grid-area: nav;
+}
+
+
+main{
+    grid-area: main;
+}
```
Comment (Jay-Gadhiya) [inline src/app.module.css:17]: maybe you can use class instead of use tag.

**Review by Jay-Gadhiya — APPROVED** [review]
Your code is good and woking fine

**Comment by mohammed-sas** [comment]
> Great work Mohammed! Just a question: Can you please explain the purpose of app.module.css? I did not clearly understand its purpose and why you used it.

in order put the components in grid that are rendered inside app component, that's the reason i had app.module.css

**Pull request merged**